extensions/amp-story/0.1/amp-story-page.js

```diff
@@ -628,12 +628,12 @@ export class AmpStoryPage extends AMP.BaseElement {
     }
 
     this.debounceToggleLoadingSpinner_(true);
-    videos.forEach(videoEl => {
+    for (let videoEl of videos) {
```

Comment:
If I remember correctly, we can't use `for...of` loops

Reply:
Discovered ``gulp pr-check``, will make sure I run it before submitting any code next time :)